doc/bookkeeper-options.md

```diff
@@ -17,3 +17,28 @@ spec:
       codahaleStatsOutputFrequencySeconds: "30"
 ...
 ```
+### Bookkeeper JVM Options
+
+It is also possible to tune the Bookkeeper JVM by passing customized JVM options. Bookkeeper JVM Options
+are obvisouly for Bookkeeper JVM whereas aforementioned BookKeeper options are for BookKeeper server configuration.
+
+The format is as follows:
+```
+...
+spec:
+  bookkeeper:
+    bookkeeperJVMOptions:
+      memoryOpts: ["-Xms2g", "-XX:MaxDirectMemorySize=2g"]
+      gcOpts: ["-XX:MaxGCPauseMillis=20"]
+      gcLoggingOpts: ["-XX:NumberOfGCLogFiles=10"]
+      # extraOpts are options that don't belong to the previous categories. If you
+      # are not sure what categories that your option belongs to, please check if your option
```

Comment:
Explanation is very confusing. There is no need to add it here inside the code snippet. Please add it later in the documentation. Perhaps we could just say "JVM options that  don;t belong to the earlier 3 categories can be mentioned under `extraOpts`"